### ISSUE TITLE
PDO documentation examples are wrong: preparedExecute

### ISSUE THREAD

**Issue by geoidesic** [phalcon/docs#528]
```
http://docs.phalconphp.com/en/latest/api/Phalcon_Db_Adapter_Pdo.html
```
All of the examples are missing the required third parameter for the 'executePrepared' method. Having examples of the third parameter would be useful so that we can figure out what format is expected in receipt.

**Comment by nathanielks** [comment]
Same as #166. @phalcon any word on this?

**Comment by SidRoberts** [comment]
Fixed in cc420b8.
